### DISCUSSION TITLE
Any X-Plane Consideration?

### DISCUSSION THREAD

**Discussion by Andywmm9** [nguyenquyhy/Flight-Tracker-StreamDeck#124] [Ideas]
I know it would be a big task, but the X-Plane network protocol is well documented.  Has this ever been considered?

**Comment by rmroc451** [comment]
@nguyenquyhy might do some prepare stuff he said. Something about a big new tubeliner coming?

I am going to focus on MSFS, I don't have P3D or Xplane :)

**Comment by nguyenquyhy** [comment]
I build this plugin mainly to serve my own use. And since MSFS release, I rarely went back to P3D and have never touched X-plane again, so I don't think I'll invest time in X-plane or P3D (although it might work with FSX and P3D with SimConnect compatibility mode).

I won't be against PR on X-plane compatibility though, but it might be difficult since the plugin is designed around SimConnect.
